cmd/reissuer/main.go

```diff
@@ -383,6 +404,89 @@ func makeCertificateProto(c *cli.Context) (err error) {
 	return nil
 }
 
+func revokeCerts(c *cli.Context) (err error) {
+	vaspID := c.String("vasp")
+	fmt.Printf("lookup up vasp with id %s\n", vaspID)
+
+	var vasp *pb.VASP
+	if vasp, err = db.RetrieveVASP(vaspID); err != nil {
+		return cli.Exit(fmt.Errorf("could not find VASP record: %s", err), 1)
+	}
+
+	// Check with the user if we should continue with the certificate revocation
+	fmt.Printf("revoking certs for %s\n", vasp.CommonName)
+	if !c.Bool("yes") {
+		if !askForConfirmation("continue with certificate revocation?") {
+			return cli.Exit(fmt.Errorf("canceled by user"), 1)
+		}
+	}
+
+	// Mark any outstanding certificate requests as rejected.
+	var certreqs []string
+	if certreqs, err = models.GetCertReqIDs(vasp); err != nil {
+		return cli.Exit(fmt.Errorf("could not get certificate request ids: %s", err), 1)
+	}
+
+	for _, crid := range certreqs {
+		var certreq *models.CertificateRequest
+		if certreq, err = db.RetrieveCertReq(crid); err != nil {
+			fmt.Printf("error retrieving certreq %s: %s\n", crid, err)
+			continue
+		}
+
+		// Only update certreqs taht are not completed.
+		if certreq.Status < models.CertificateRequestState_COMPLETED {
+			if err = models.UpdateCertificateRequestStatus(
+				certreq,
+				models.CertificateRequestState_CR_REJECTED,
+				"",
```

Comment:
Do we want to put in a tag here so that we can tell from the audit log that this was not automated and instead we rejected it from the script?

Reply:
```suggestion
				"certificate request canceled by admin",
```